### DISCUSSION TITLE
How many databases can I have in the scaler tier?

### DISCUSSION THREAD

**Discussion by blntrsz** [planetscale/discussion#209] [Q&A] [answered]
In the free tier I can only have one database. I'm thinking about upgrading because I have plenty of pet project which would need small databases. So my question would be, how many databases can I create in the Scaler tier?

**Comment by jgreet** [comment, marked as answer]
Scaler plans are per database, you can have as many Scaler databases as you'd like under a single organization but each one is on a separate $29/month plan.

Each organization can also have one free tier database at a time.

**Reply by devDharma** [reply]
Our app has multi tenant architecture and each tenant have their own DB. Not all will be using the same amount as others. If it is 29$ per DB then it is not viable for our app. 
Any thoughts?

**Reply by jgreet** [reply]
It sounds like a host that supports multiple databases in a single MySQL instance would be a better fit for this application.

**Reply by pratikkuikel** [reply]
@devdharma Hey there, I do have the same need. What service are you using currently to host all the tenants DBs ? What do you think would be a better fit for this situation?

**Reply by r14ull** [reply]
I'm thinking of an EC2 instance for mysql but wanting to separate the concerns incase one customer decides to overload their database bringing down others. Looking for others insight also would be great.

**Reply by maulik9211** [reply]
I also have the same need. I am thinking about Linode managed MySQL or AWS RDS to handle the backup, uptime, security and then in that one "Instance" host all the "Databases" for individual companies. Additionally, a new user will be created for each company with priviledges to ONLY access their company database.

**Reply by devDharma** [reply]
I'm now using AWS RDS and spin new RDS instance if one becomes too loaded.

**Reply by maulik9211** [reply]
Is it manually deployed or automatically based on the usage metrics?
